go-controller/pkg/ovn/gateway_init.go

```diff
@@ -561,7 +561,7 @@ func (oc *Controller) addExternalSwitch(prefix, interfaceID, nodeName, gatewayRo
 	// Connect GR to external_switch with mac address of external interface
 	// and that IP address. In the case of `local` gateway mode, whenever ovnkube-node container
 	// restarts a new br-local bridge will be created with a new `nicMacAddress`.
-	externalRouterPort := types.GWRouterToExtSwitchPrefix + gatewayRouter
+	externalRouterPort := prefix + types.GWRouterToExtSwitchPrefix + gatewayRouter
```

Comment:
nice catch! depending on whether this was called from 1br or 2br we'd have "" or "exgw"...